src/mlpack/methods/lsh/lsh_main.cpp

```diff
@@ -137,6 +139,7 @@ int main(int argc, char *argv[])
   const size_t numProj = CLI::GetParam<int>("projections");
   const size_t numTables = CLI::GetParam<int>("tables");
   const double hashWidth = CLI::GetParam<double>("hash_width");
+  const size_t numProbes = CLI::GetParam<int>("num_probes");
```

Comment:
Do you need to do an explicit cast here to avoid a warning?  I am not sure, I think it depends on the compiler, but usually I will do something like `(size_t) CLI::GetParam<int>("num_probes");`.